fastify.js

```diff
@@ -337,6 +338,8 @@ function build (options) {
   }
 
   function inject (opts, cb) {
+    if (!shot) return cb(new Error('"shot" library is not installed: "npm install shot --save-dev"'))
```

Comment:
It is not compliant with the callback javascript convention.

Should I change all tests and doc?